test/execution/server_test.exs

```diff
@@ -105,6 +112,10 @@ defmodule Wanda.Execution.ServerTest do
 
       assert_receive :executed
       assert_receive {:DOWN, ^ref, _, ^pid, :normal}
+
+      history_log = Repo.all(ExecutionResult)
+      assert 1 == length(history_log)
```

Comment:
this assert is not needed because of the pattern match above

Reply:
You mean below?

Reply:
yes :D